### DISCUSSION TITLE
v1.7.0 Prerelease 4

### DISCUSSION THREAD

**Discussion by iwalton3** [jellyfin/jellyfin-desktop#229] [General]
This is another prerelease for testing. It includes the following additional changes:

- Update web client to 10.8.0 Beta 1.
- The Windows builds are now built using `windows-2022`.

Please note as this is a prerelease the macOS build is not notarized. I have manually smoke tested the Windows and Debian Bullseye versions.

<hr /><em>This discussion was created from the release <a href='https://github.com/jellyfin/jellyfin-media-player/releases/tag/v1.7.0-pre4'>v1.7.0 Prerelease 4</a>.</em>

**Comment by ghost** [comment]
I can not install it on mac 10.15.7, it says that it is only compatible with version 11.6 or later.
Will version 1.7 be able to be installed only on mac os 11?

**Reply by alexmc6** [reply]
same here.

**Reply by iwalton3** [reply]
Looks like it might be because CI is now running on Mac OS 11. https://github.com/jellyfin/jellyfin-media-player/runs/5730987110?check_suite_focus=true

**Comment by ruchid123123** [comment]
从https://www.guojiangbo.com/index.php/2021/05/31/%e4%b8%80%e4%b8%aa%e5%8f%af%e4%bb%a5%e6%9c%ac%e5%9c%b0%e8%a7%a3%e7%a0%81%e7%9a%84jellyfin%e5%ae%a2%e6%88%b7%e7%ab%af/
来的

**Comment by sambartik** [comment]
Will there be another media player release to update to the current jellyfin web client: 10.8.0 Beta 2?

**Reply by iwalton3** [reply]
I don't see anything ground-breaking in the diff, so I probably won't unless someone needs it. https://github.com/jellyfin/jellyfin-web/compare/v10.8.0-beta1...v10.8.0-beta2

**Reply by sambartik** [reply]
Oh, yeah, it really doesnt look like a lot have changed. However, could you plesse take a look at an issue over at jellyfin/jellyfin-plugin-trakt#129?

**Comment by Spiritreader** [comment]
Is the quality adjustment / subtitle offset menu gone for good / on purpose?
I thought it was quite useful to change the quality without leaving the media file and changing the default under "Playback"
![image](https://user-images.githubusercontent.com/18438899/170885973-51ca0c1a-6d95-48b0-8ea9-1b3fc662b5a2.png)

**Reply by iwalton3** [reply]
I didn’t remove it. It sounds like breakage from upstream.

**Reply by iwalton3** [reply]
So I am testing this preparing for a release and I still have a quality selector.

**Reply by Shadowghost** [reply]
Most likely fixed by https://github.com/jellyfin/jellyfin-web/pull/3668

**Comment by JumpyJim404** [comment]
Everything works perfectly using Jellyfin android phone app to cast to media player installed on my Windows PC.  I can't see a way to alter colour correction in either the Windows player or android app, as it would be useful to be able to control the brightness/contrast/hue/saturation settings directly within the player.

Brilliant application.
